<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomizeWin</class>
 <widget class="QWidget" name="CustomizeWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>334</height>
   </rect>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/appLogo/images/favicon.ico</normaloff>:/appLogo/images/favicon.ico</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="labelLimitedV">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>60</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="res.qrc">:/appLogo/images/xiyou.webp</pixmap>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="appName">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>人物角色联络分析系统</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>88</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton_2">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/buttonLogo/images/customize.png</normaloff>:/buttonLogo/images/customize.png</iconset>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="menuSetup">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/appLogo/images/setup3d.png</normaloff>:/appLogo/images/setup3d.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="popupMode">
         <enum>QToolButton::InstantPopup</enum>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnExit">
        <property name="styleSheet">
         <string notr="true">background-color: qradialgradient(spread:repeat, cx:0.5, cy:0.5, radius:0.077, fx:0.5, fy:0.5, stop:0 rgba(255, 240, 0, 150), stop:0.497326 rgba(237, 171, 142, 147), stop:1 rgba(247, 255, 93, 147));</string>
        </property>
        <property name="text">
         <string>退出</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Home" name="widget" native="true"/>
   </item>
  </layout>
  <action name="action_novel">
   <property name="icon">
    <iconset>
     <normalon>:/buttonLogo/images/c_yellow.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>小说管理和配置</string>
   </property>
   <property name="toolTip">
    <string>小说管理和配置</string>
   </property>
  </action>
  <action name="action_minimum">
   <property name="text">
    <string>最小化</string>
   </property>
  </action>
  <action name="action_resize">
   <property name="text">
    <string>窗口还原</string>
   </property>
   <property name="toolTip">
    <string>窗口还原</string>
   </property>
  </action>
  <action name="action_maximum">
   <property name="text">
    <string>最大化</string>
   </property>
   <property name="toolTip">
    <string>最大化</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Home</class>
   <extends>QWidget</extends>
   <header>home.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
